### PR TITLE
Restructure and rename to make this repo consistent with other RockLib repos

### PR DESCRIPTION
**Pull request by JustinMoss**

### CODE REVIEW

File: RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard1.6;net45;</TargetFrameworks>
```
Comment (bfriesen) [inline RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj]: For .net framework, let's target 4.0 and 4.5.1

File: RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>RockLib.EmbeddedNativeLibrary</PackageId>
+    <PackageVersion>1.1.2</PackageVersion>
```
Comment (bfriesen) [inline RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj]: Should make the version the same as the current tag (2.0.2).

File: RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <Authors>Quicken Loans</Authors>
+    <Description>Consume native libraries from .NET by adding as embedded resources.</Description>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageReleaseNotes></PackageReleaseNotes>
+    <Copyright>Copyright 2017 (c) Quicken Loans Corporation. All rights reserved.</Copyright>
+    <PackageTags>rocklib embedded native library</PackageTags>
+    <Version>1.1.0</Version>
```
Comment (bfriesen) [inline RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj]: Can we change this to 2.0.0?

File: RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>RockLib.EmbeddedNativeLibrary</PackageId>
+    <PackageVersion>1.1.2</PackageVersion>
+    <Authors>Quicken Loans</Authors>
+    <Description>Consume native libraries from .NET by adding as embedded resources.</Description>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageReleaseNotes></PackageReleaseNotes>
+    <Copyright>Copyright 2017 (c) Quicken Loans Corporation. All rights reserved.</Copyright>
```
Comment (bfriesen) [inline RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj]: Copyright years should be 2015-2018.

**Review by bfriesen — CHANGES_REQUESTED** [review]
A few things in the csproj.

File: RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj (hunk 2 above)
Comment (JustinMoss) [reply RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj]: 👍

File: RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj (hunk 3 above)
Comment (JustinMoss) [reply RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj]: 👍

File: RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj (hunk 4 above)
Comment (JustinMoss) [reply RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj]: 👍

File: RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj (hunk 1 above)
Comment (JustinMoss) [reply RockLib.EmbeddedNativeLibrary/RockLib.EmbeddedNativeLibrary.csproj]: This causes an issue due to GetTypeInfo() and IReadOnlyCollection not being available before 4.5.  To change this I would need to add constant bases conditionals in at least a few places.

**Review by bfriesen — APPROVED** [review]
👍

**Pull request merged**